Makefile

```diff
@@ -1,14 +1,24 @@
 # See our wiki for getting the CalculiX dependencies:
```

Comment:
You could also merge `master` to `v2.17` or cherry-pick the respective commit, to avoid this duplicate change.

Reply:
Yes, ideally `v2.17` should have been fully updated before we merge this. I asked @KyleDavisSA about it but I'm not confident to do that merge myself.

Reply:
Why not? Let me know if I can help.

Reply:
Do we want to rebase on develop first?

Reply:
I think yes. Otherwise the duplication of the Makefile would stay for a while.
(Then I don't know if I should change the target branch of the PR, or open a new one on the rebased after fixing locally the conflicts ?)

Reply:
Done, there is no `Makefile` here anymore.